src/sparse/impl/KokkosSparse_spgemm_mkl2phase_impl.hpp

```diff
@@ -169,7 +169,7 @@ void mkl2phase_symbolic(
     }
 #endif
 
-#if __INTEL_MKL__ >= 2018
+#if __INTEL_MKL__ == 2018 && __INTEL_MKL_UPDATE__ == 2
```

Comment:
I believe greater than or equal to 2 should be fine here for now. If something goes wrong in 3 we will come back and fix it.

Reply:
@srajama1 I don't mind changing back to >=, but thought it would be helpful for earlier/easier detection of mkl api changes to force an error of intel/18.2.xyz is not used, since the api changed not just between version (17 to 18) but also updates within a version (update 0 and 1 to 2, for example). If we error out with untested versions we can verify early on if the api changed, and based on intel 18 experience this seems quite possible.